test/unit/mc/test_cholesky.cpp

```diff
@@ -0,0 +1,152 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#include "dlaf/mc/cholesky_local.h"
+
+#include <typeinfo>
```

Comment:
What symbol(s) do you need from `<typeinfo>`?

Reply:
You're right. I don't need it